recipes/telluric/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "telluric" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "88c92f26b75d3600d4546a533292a13ceb38a37e93e627a232b72308075fd0a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.4
+    - setuptools
+    - pip
+
+  run:
+    - python >=3.4
+    - affine
+    - dsnparse
+    - fiona >=1.7.2,<2.*
+    - folium
+    - ipyleaflet
+    - pyproj
+    - shapely >=1.6.3,<2.*
+    - rasterio >=1.0,<2.*
+    - pillow
+    - mercantile >=0.10.0
+    - matplotlib
+    - boltons
+    - imageio
+
+test:
+  imports:
+    - telluric
+
+about:
+  home: http://github.com/satellogic/telluric
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
The ``LICENSE`` file is not being packaged.

Reply:
I opened a PR for that, please see
https://github.com/satellogic/telluric/pull/147

Reply:
Thank you @marcelotrevisani! Merged.

Reply:
@drnextgis , is it possible to generate a tag with this modification?

Reply:
I have updated PR with new version.